sphinx/texinputs/Makefile_t

```diff
@@ -82,11 +87,14 @@ xz: tar
 {%- endif %}
 	$(PDFLATEX) $(LATEXOPTS) "$<"
 
+{%- if latex_engine != 'xelatex' %}
+
 %.ps: %.dvi
-	dvips '$<'
+	dvips "$<"
+{%- endif %}
 
 clean:
-	rm -f *.log *.ind *.aux *.toc *.syn *.idx *.out *.ilg *.pla *.ps *.tar *.tar.gz *.tar.bz2 *.tar.xz $(ALLPDF) $(ALLDVI) *.fls *.fdb_latexmk
+	rm -f *.log *.ind *.aux *.toc *.syn *.idx *.out *.ilg *.pla *.ps *.tar *.tar.gz *.tar.bz2 *.tar.xz $(ALLPDF) $(ALLDVI) $(ALLXDV) *.fls *.fdb_latexmk
```

Comment:
I am much confused here. This clean rule will not work on Windows right ?

I must admit I have absolutely no idea how Makefiles run on Windows. I see mention on the net of things like MSYS and MinGW, but what kind of shell is that ?

Currently I removed a `rm -f %D;` in latexmkjarc because the command line needs `del` on Windows.

Perhaps Makefile enablers on Windows translate a `rm`  into `del` so the Makefile has no such restriction while the latexmk config file latexmkrc however does have. Lost in the agonies of ignorance ...